fate/src/test/java/org/apache/accumulo/fate/zookeeper/ZooCacheTest.java

```diff
@@ -80,9 +80,9 @@ private void testGet(boolean fillStat) throws Exception {
     if (fillStat) {
       myStat = new ZcStat();
     }
-    long now = System.currentTimeMillis();
+    final long ephemeralOwner = 123456789l;
```

Comment:
Using Random here might make this more clear that this number doesn't matter... we're just using it to ensure the number we put in is the same one we see later. But, a brief comment to that would also work.